vies/fields.py

```diff
@@ -31,9 +31,10 @@ def compress(self, data_list):
             value = ''.join(data_list)
             try:
                 vatin = VATIN(*data_list)
+                is_valid = vatin.is_valid()
             except ValueError as e:
```

Comment:
Do you really want to catch a ValueError, or do you want to catch any other exception=?